package.json

```diff
@@ -6,7 +6,10 @@
   "scripts": {
     "start": "node index.js",
     "lint": "eslint .",
-    "test": "jest"
+    "lint:fix": "npm run lint -- --fix",
```

Comment:
Either use `npm run` approach in both script `lint:fix` and `test:watch`/`test:coverage` or simply call the original command in both cases but code must be consistent along the file.